components/common/WalletModal.vue

```diff
@@ -140,6 +141,13 @@ export default class WalletModal extends Vue {
     }
   }
 
+  protected formatAccount(account: WalletAccount): WalletAccount {
+    return {
+      ...account,
+      address: formatAddress(account.address, this.ss58Format),
```

Comment:
Wouldn't it be better to move this logic to `subscribeAccounts` & `getAccounts` properties in `BaseDotsamaWallet` file